src/view-components/side-menu/side-menu.component.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect } from "react";
+import { SideNav } from "carbon-components-react/es/components/UIShell";
+import { SideNavProps } from "carbon-components-react";
+import { ExtensionSlot, useLayoutType } from "@openmrs/esm-framework";
+import styles from "./side-menu.component.scss";
+import { isTablet } from "../../utils/sidebar-utils";
+
+interface SideMenuPanelProps extends SideNavProps {}
+
+const LAYOUT_CSS_CLASS = "omrs-sidenav-expanded";
+
+const SideMenuPanel: React.FC<SideMenuPanelProps> = () => {
+  const layout = useLayoutType();
+
+  useEffect(() => {
```

Comment:
so this effect append or removes the expanded class on the body. The expanded class adds a margin left to account for the sidebar in order to achieve the sidebar / main content layout for desktop.

When the component is dismounted, we just want to remove the class because no other page uses the sidebar for desktop